### DISCUSSION TITLE
Why I don't see beanio3 details in official website http://beanio.org/

### DISCUSSION THREAD

**Discussion by josephmicheal** [beanio/beanio#154] [Q&A] [answered]
Hi,

I am planning to use beanio for my upcoming project but as per official website only beanio2.1 details are there and provided download link goes to https://code.google.com/archive/p/beanio/downloads where last update was done in 2013.

Please clarify why I don't see beanio3 in official website. Is there any diff between this github repo and official website? 

Thanks.

**Comment by bjansen** [comment, marked as answer]
Hi,

The original maintainer of BeanIO has been unreachable for years, so this project is a fork with a new website (nobody has access to modify beanio.org).

**Reply by josephmicheal** [reply]
Thanks for the response.
